modules/tile-converter/src/i3s-converter/helpers/geometry-converter.js

```diff
@@ -158,7 +161,7 @@ function convertAttributes(tileContent) {
     });
   }
 
-  const nodes = tileContent.gltf.scene.nodes;
+  const nodes = (tileContent.gltf.scene || tileContent.gltf.scenes?.[0] || tileContent.gltf).nodes;
```

Comment:
1. This doesn't look correct. IIRC `gltf.scene` is an index into the scene array.
2. Easier to read if we split such logic on multiple lines
3. Good if we can add a link to an "offending" tileset in the comment.

```suggestion
  const sceneIndex = tileContent.gltf.scene || 0;
  const scene = tileContents.gltf.scenes?.[sceneIndex];
  // Note: Some tilesets are not glTF conformant and do not have a `scenes` array, just `nodes`
  const nodes = scene ? scene.nodes : tileContents.gltf.nodes;
```

Reply:
Our GLTF postprocessing replaces this index to real object